script/main.py

```diff
@@ -282,7 +282,31 @@ def users_born(conn, start_date, end_date):
     full_date_of_birth = datetime.strptime(user_date_birth, '%Y-%m-%dT%H:%M:%S.%fZ')
     date_of_birth = full_date_of_birth.date()
     print(f'User {user_name} {user_last_name} was born in {date_of_birth}')
-  
+
+
+def most_secure_password(conn):
+  passwords_list =  [i[2] for i in select_all_gender_and_dob_date_and_password(conn)]  
+  password_and_score_tuple_list = []
+  for password in passwords_list:
+    total = 0
+    if len(password) >= 8:
+      total += 5      
+    if any(letter.isupper() for letter in password):
+      total += 2
+    if any(letter.islower() for letter in password):
+      total += 1
+    if any(letter.isdigit() for letter in password):
+      total += 1
+    if any(not letter.isalnum() for letter in password):
+      total += 3
+
+    password_and_score_tuple_list.append(tuple((password, total)))
```

Comment:
Instead of:
```python
tuple((password, total))
```
write
```python
(password, total)
```

Reply:
I cannot do this because I am getting an error:
`
TypeError: append() takes exactly one argument (2 given)
`

Reply:
Try:
```python
password_and_score_tuple_list.append((password, total))
```

Reply:
it works